pkg/cmd/admin/router/router.go

```diff
@@ -438,7 +440,8 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 			return fmt.Errorf("router could not be created; you must specify a service account with --service-account")
```

Comment:
This is actually wrong - if you're not using hostNetwork (it's false) you don't need a service account.